kinesis-test/src/main/java/com/hazelcast/jet/tests/kinesis/KinesisTest.java

```diff
@@ -85,7 +85,11 @@ protected void init(JetInstance client) {
     @Override
     protected void teardown(Throwable t) {
         for (String streamName : streamNames) {
-            helper.deleteStream(streamName);
+            try {
+                helper.deleteStream(streamName);
+            } catch (Exception e) {
+                logger.severe("Exception when deleting stream: " + streamName, e);
```

Comment:
I would suggest to fail the test after all streams tried to be deleted and any of them was not deleted. In that case we need to delete them manually. Test failure should inform us about that, I think we can miss it if it is only logged. However part of test failure should be an information that this particular exception does not mean failed soak test it just means that some streams cannot be deleted and should be deleted manually. We should also put there an info that the rest of test log should be checked whether there is also some test failure.